ProcessLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -134,6 +134,8 @@ struct SingleProcessData
     //! cast of \c tdisc_ode_sys to NumLib::InternalMatrixStorage
     NumLib::InternalMatrixStorage* mat_strg = nullptr;
 
+    int equation_id = 0;
```

Comment:
Docu.

Still not clear to me is the difference btw. an equation id and a process id, could you explain, please?

Reply:
Documented, and changed it to `process_id`.